src/fidesops/core/config.py

```diff
@@ -178,14 +178,17 @@ class FidesopsConfig(FidesSettings):
     execution: ExecutionSettings
 
     is_test_mode: bool = os.getenv("TESTING") == "True"
-    hot_reloading: bool = os.getenv("DEV_MODE") == "True"
+    hot_reloading: bool = os.getenv("FIDESOPS_HOT_RELOAD") == "True"
+    dev_mode: bool = os.getenv("FIDESOPS_DEV_MODE") == "True"
 
     class Config:  # pylint: disable=C0115
         case_sensitive = True
```

Comment:
I think we'll need to add `env_prefix = FIDESOPS_` for this to work being able to reference these vars as `FIDESOPS_VAR_NAME`. We should use `FIDESOPS__` as the prefix though, since that's consistent with the sub-sections above.

Reply:
Added an extra underscore, I was following the FIDESOPS_CONFIG_PATH variable, so I've updated that to also have double underscores.